File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Convert Word document to PDF in ASP.NET MVC | Syncfusion
+description: Convert Word document to PDF without Microsoft Word or interop dependencies in ASP.NET MVC application using .NET Word (DocIO) library.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word document to PDF in ASP.NET MVC
+
+Syncfusion Essential DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **convert a Word document to PDF in ASP.NET MVC**.
+
+## Steps to convert word document to PDF in C#:
+
+Step 1: Create a new ASP.NET Web Application  project.
+
+![Create ASP.NET MVC application in Visual Studio](ASP-NET-MVC_images/CreateProjectforConversion.png)
+
+Step 2: Select the MVC application
+
+![Create ASP.NET MVC application in Visual Studio](ASP-NET-MVC_images/MVC.png)
+
+Step 3: Install the [Syncfusion.DocIO.AspNet.Mvc5](https://www.nuget.org/packages/Syncfusion.DocIO.AspNet.Mvc5) and [Syncfusion.DocToPdfConverter.AspNet.Mvc5](https://www.nuget.org/packages/Syncfusion.DocToPdfConverter.AspNet.Mvc5) NuGet package as a reference to your project from [NuGet.org](https://www.nuget.org/).
```

Comment:
Installing DocToPDFConverter NuGet is enough, it install the DocIO as dependent

Reply:
Updated sir